setup.py

```diff
@@ -0,0 +1,44 @@
+import setuptools
+
+# Commands to publish new package:
+#
+# rm -rf dist/
+# python setup.py sdist
+# twine upload dist/*
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="ccc-coef",
+    version="0.1.5",  # remember to change libs/ccc/__init__.py file also
```

Comment:
Is it typical that you have to change the version number in so many different places? You did a good job of documenting this, just wondering if it's normal since I haven't put a package on PyPI like this.

Reply:
Great question, I had the same one. Not sure if it's normal, but I also think it's sort of too many places (more than 1) where you have to change the version. I don't think modifying `__init__.py` is really necessary though. I was following a tutorial that suggested to install a separate package to keep all those places synchronized but I won't install a package just for that :)